src/about.njk

```diff
@@ -125,12 +125,12 @@
         <p class="text-muted">Birmingham leader</p>
         <ul class="list-inline social-buttons">
           <li class="list-inline-item">
-            <a href="https://github.com/CFin86" target="_blank" class="shadow-sm">
+            <a href="https://github.com/CFin86" target="_blank" class="shadow-sm" aria-label="Navigation link to Chris Finney GitHub Page">
```

Comment:
Screen readers will announce this as a link, so a better aria-label would be "Chris Finney's GitHub Page"